app/_src/reference/kubernetes-annotations.md

```diff
@@ -605,6 +605,14 @@ spec:
         [...]
 ```
 
+{% if_version gte:2.4.x %}
+### `kuma.io/wait-for-dataplane-ready`
+
+Define if you want the init container to wait for the dataplane to be ready before exiting.
```

Comment:
Not sure if this is correct. It's more `kuma-sidecar` container waiting for DP to be ready and when ready application container starts